protostar/commands/test/test_command.py

```diff
@@ -75,6 +75,13 @@ def arguments(self) -> List[Command.Argument]:
                 description="Compile as account contract.",
                 type="bool",
             ),
+            Command.Argument(
+                name="disable-hint-validation",
+                description=(
+                    "Disable hint validation in contracts declared by the `declare` cheatcode.\n"
```

Comment:
Can we add information about `deploy_contract` too? Probably `deploy_contract` is more popular cheatcode than declare.